### DISCUSSION TITLE
Création de versions

### DISCUSSION THREAD

**Discussion by spelhate** [megalis-bretagne/opendata-marqueblanche#9] [Ideas]
Bonjour

Il pourrait être intéressant de créer côté Github une première release (version) de l'application. Ceci nous permettrait pour les demandes d'évolutions de cibler les versions mineures et majeures suivantes.
Qu'en pensez vous @lvinsonneau @yguenneugues @clemoigno

**Comment by lvinsonneau** [comment]
+1 !
